src/applications/discharge-wizard/helpers/index.jsx

```diff
@@ -392,9 +395,10 @@ export const determineBoardObj = (formResponses, noDRB) => {
   return boardObj;
 };
 
+// Determines form specific data based on form responses.
```

Comment:
I think "form" could be confused here for the DUW form (the `/discharge-upgrade-instructions/introduction` form) rather than the physical forms that Veterans need to fill out from the results pages. Not sure; thoughts?